certified-connectors/AXtension Content Gate/apiDefinition.swagger.json

```diff
@@ -0,0 +1,467 @@
+                            "parameter": "connections.providerReferenceId"
+                          }
+                        }
+                      },
+                      "x-ms-dynamic-list": {
+                        "operationId": "List-Business-Entity-Types",
+                        "itemValuePath": "provider/externalType",
+                        "itemTitlePath": "name",
+                        "parameters": {
+                          "providerReferenceId": {
+                            "parameterReference": "body/connections/providerReferenceId"
+                          }
+                        }
+                      }
+                    },
+                    "externalId": {
+                      "type": "string",
+                      "description": "The identifier that uniquely identifies the business entity.",
+                      "title": "Business Entity Identifier",
+                      "x-ms-visibility": "important"
+                    }
+                  },
+                  "description": "connections",
+                  "required": [
+                    "externalId",
+                    "externalType",
+                    "providerReferenceId"
+                  ]
+                },
+                "userProperties": {
+                  "$ref": "#/definitions/UserProperties"
+                }
+              },
+              "required": ["assets", "category", "connections"]
+            }
+          }
+        ]
+      }
+    },
+    "/api/ContentCategories": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "id": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "The Content Category identifier.",
+                    "title": "Id"
+                  },
+                  "name": {
+                    "type": "string",
+                    "description": "The name of the Content Category.",
+                    "title": "Name"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "List Content Categories",
+        "operationId": "List-Content-Categories",
+        "description": "Lists all configured content categories.",
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/api/ContentEntityProperties/schema": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "schema": {
+                  "type": "object"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Content Entity properties schema.",
```

Comment:
@JoepKillaars Please remove the period at the end of the summary value.
Refer - [Documenation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#submission-requirements)
`operation descriptions should be full, descriptive sentences, and end in punctuatio`

Reply:
Removed the period.